zulipterminal/api_types.py

```diff
@@ -85,6 +85,24 @@ class Subscription(TypedDict):
     stream_weekly_traffic: Optional[int]
 
 
+class RealmUserData(TypedDict):
```

Comment:
This is an API response for a general realm user. I tried separating parameters unique to cross-realm bots, but found almost all parameters overlapping to the general API response. Let me know if we could further distinguish this TypedDict.